app/javascript/plugins/swipable.js

```diff
@@ -0,0 +1,70 @@
+function moveToSelected(element) {
+  if (element == "next") {
+    var selected = $(".selected").next();
+  } else if (element == "prev") {
+    var selected = $(".selected").prev();
+  } else {
+    var selected = element;
+  }
+
+  var next = $(selected).next();
+  var prev = $(selected).prev();
+  var prevSecond = $(prev).prev();
+  var nextSecond = $(next).next();
+
+  $(selected)
+    .removeClass()
+    .addClass("selected");
+
+  $(prev)
+    .removeClass()
+    .addClass("prev");
+  $(next)
+    .removeClass()
+    .addClass("next");
+
+  $(nextSecond)
+    .removeClass()
+    .addClass("nextRightSecond");
+  $(prevSecond)
+    .removeClass()
+    .addClass("prevLeftSecond");
+
+  $(nextSecond)
+    .nextAll()
+    .removeClass()
+    .addClass("hideRight");
+  $(prevSecond)
+    .prevAll()
+    .removeClass()
+    .addClass("hideLeft");
+}
+
+// Eventos teclado
+$(document).keydown(function(e) {
+  switch (e.which) {
+    case 37: // left
+      moveToSelected("prev");
+      break;
+
+    case 39: // right
+      moveToSelected("next");
+      break;
+
+    default:
+      return;
+  }
+  e.preventDefault();
+});
+
+$("#carousel div").click(function() {
+  moveToSelected($(this));
+});
+
+$("#prev").click(function() {
+  moveToSelected("prev");
+});
+
+$("#next").click(function() {
+  moveToSelected("next");
+});
```

Comment:
Same code as in application.js? Does it need to be written down twice?Export Import?

Reply:
The 'Eventos teclado' should remain in the application.js page